PlexAPI/PlexAPI.csproj

```diff
@@ -1,10 +1,10 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <IsPackable>true</IsPackable>
-    <PackageId>PlexAPI</PackageId>
+    <PackageId>JasonLandbridge.PlexAPI.SDK</PackageId>
```

Comment:
@JasonLandbridge What should be the name of the Nuget package?

I tried `PlexAPI` but that already exists.

It has to be unique which you can verify here: https://www.nuget.org/packages